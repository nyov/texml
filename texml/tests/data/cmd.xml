<TeXML>
<cmd name="documentclass">
  <opt>12pt</opt>
  <parm>letter</parm>
</cmd>
<cmd name="par"/>
<cmd name="o">
	<opt>o1</opt>
</cmd>
<cmd name="p">
	<parm>p1</parm>
</cmd>
<cmd name="op">
	<opt>o1</opt>
	<parm>p1</parm>
	<opt>o2</opt>
	<parm>p2</parm>
</cmd>
<cmd name="unusual">
	<parm>a<ctrl ch=" "/>a</parm>
	<opt>b<spec cat="esc"/>b</opt>
</cmd>
</TeXML>
