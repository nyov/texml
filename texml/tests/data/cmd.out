%
\documentclass[12pt]{letter}
\par 
\o[o1]
\p{p1}
\op[o1,o2]{p1,p2}
\unusual[b\b]{a\ a}
